### PR TITLE
COMMERCE-9337

### PR DESCRIPTION
**Pull request by baharturk**
https://issues.liferay.com/browse/COMMERCE-9337

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCart.testcase [hunk 1]
```diff
@@ -346,6 +346,106 @@ definition {
...
+			CommerceEntry.gotoMenuTab(menuTab = "Subscription");
+
+			Click(locator1 = "CommerceEntry#DELIVERY_SUBSCRIPTION_TOGGLE");
+
+			Button.clickPublish();
+
+			Alert.viewSuccessMessage();
+		}
+		
+		task ("When the product with description is added to the cart") {
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCart.testcase:382]: the description is not relevant for the test, we can omit that

Comment (baharturk) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceMiniCart.testcase:382]: yes, it should be subscription, right, thanks!

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@baharturk Well done, I left some comments. Can you take a look at them? Thanks

**Comment by baharturk** [comment]
ci:stop

**Review by alessiorendina — APPROVED** [review]

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:stop

**Comment by baharturk** [comment]
ci:stop

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**